package.json

```diff
@@ -159,7 +159,7 @@
     "settings": {
       "import/resolver": {
         "webpack": {
-          "config": "./internals/webpack/webpack.dev.babel.js"
+          "config": "./internals/webpack/webpack.prod.babel.js"
```

Comment:
Okay, I just read your comment on the original issue, should have read that first lol. So basically this works but we don't know why? I certainly cannot figure out why either #webpackmagic